pages/contact.tsx

```diff
@@ -26,6 +28,7 @@ export default function ContactPage() {
   const [toSend, setToSend] = useState<FormData>(initialState);
   const [formError, setFormError] = useState<string>('');
   const [formErrors, setFormErrors] = useState<FormData>(initialState);
+  const [formSuccess, setFormSuccess] = useState<string>('');
```

Comment:
```suggestion
  const { makeRequest, isSuccess, errorMessage } = useSubmitAWSForm();

  useEffect(() => {
    if (isSuccess) setFormError('');
    if (errorMessage) setFormError(errorMessage);

  }, [isSuccess, errorMessage]);
```
If we create a custom hook, I believe that we dont need this anymore.